### PR TITLE
setup: opt -> opts and config -> opts, remove controlPort variable

### PR DESCRIPTION
**Pull request by ralphtheninja**
Closes https://github.com/airtap/airtap/issues/96

### CODE REVIEW

File: lib/setup.js [hunk 1]
```diff
@@ -58,7 +56,7 @@ function setupTestServer (opt, cb) {
       return function (req, res) {
         var args = [].slice.call(arguments)
         if (isControlRequest(req)) {
-          args.push({ target: localUrl(controlPort) })
+          args.push({ target: localUrl(opts.control_port) })
```
Comment (yeskunall) [inline lib/setup.js]: Following what `standard` enforces, shouldn't that be camelCase?

File: lib/setup.js [hunk 2]
```diff
@@ -21,25 +21,23 @@ function setupTestServer (opt, cb) {
...
     supportServer = server
-    var config = opt
-    var loopback = config.loopback || 'localhost'
-    var controlPort = opt.control_port
+    var loopback = opts.loopback || 'localhost'
 
     var supportPort
     if (supportServer) {
-      supportPort = config.support_port = supportServer.port
+      supportPort = opts.support_port = supportServer.port
```
Comment (yeskunall) [inline lib/setup.js]: Following what `standard` enforces, shouldn't that be camelCase?

File: lib/setup.js (hunk 1 above)
Comment (ralphtheninja) [reply lib/setup.js]: Not on object properties.

Comment (yeskunall) [reply lib/setup.js]: Ah, interesting.

Comment (ralphtheninja) [reply lib/setup.js]: Personally I don't like it, but I just refactored the code.

Comment (ralphtheninja) [reply lib/setup.js]: I'm sure @feross can weigh in more. I think enforcing it on object properties would be problematic, if not stupid :)

**Pull request merged**